packages/cli/src/utils/getDependencyVersion.ts

```diff
@@ -0,0 +1,22 @@
+import * as path from 'path';
+import * as fs from 'fs';
+
+export function getDependencyVersion(folderPath: string, dependencyName: string, fileSystem = fs) {
```

Comment:
you could have made use of nodes require mechanism

```
return require('react/package.json')?.version
```

Reply:
this should always find the actual used react package

Reply:
No, this is about the react version

Reply:
And it will most probably find the react instance of the cli

Reply:
right, we also have execution path in our config